### DISCUSSION TITLE
When will FEMU support OCSSD standard 2.0?

### DISCUSSION THREAD

**Discussion by lovers-under-moon** [MoatLab/FEMU#46] [General]
Hi,I want to run SPDK on FEMU, but the libftl of SPDK only supports the OCSSD standard 2.0.And currently FEMU only supports OpenChannel Specification 1.2, the newer 2.0 spec support in work-in-progress and will be added soon.I can run SPDK in QEMU,but the performance is bad.That would be a great thing if there are an OCSSD standard 2.0 support in FEMU.

**Comment by huaicheng** [comment]
Per your request, I have added OCSSD 2.0 support in FEMU. Check out the latest commits and README for how to run it.

I didn't test OCSSD 2.0 under SPDK though, please let me know if you find any bugs in doing so. Patches for fixes are also very welcome!

Thanks!
